sections/register-form.liquid

```diff
@@ -358,6 +376,11 @@
         "id": "signup_label",
         "label": "Sign up button label"
       },
+      {
+        "type": "text",
+        "id": "button_label",
```

Comment:
Similar to the other PR. I think `button_label` is a bit too vague. Can it be more descriptive so we know that this is for the accept terms & conditions button?